getting-started.md

```diff
@@ -72,13 +70,13 @@ If everything works, you should see your starter app on your device or simulator
 <div id="starter-app-screenshots">
   <div class="box2">
     <figure>
-      <img src="/images/flutter-starter-app-ios.png">
+      <img src="/images/flutter-starter-app-ios.png" alt="iOS screenshot">
```

Comment:
https://html.spec.whatwg.org/#a-key-part-of-the-content in particular has examples of screen shot alt texts